pkgs/development/tools/hindsight/default.nix

```
@@ -0,0 +1,66 @@
+{ pkgs, lib, fetchFromGitHub }:
```

```suggestion
{ lib, fetchFromGitHub }:
```

You should never add pkgs to inputs.

---

FYI, there are a couple examples of passing pkgs as inputs in the python language doc you linked: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#handling-dependencies